dataactbroker/handlers/fileHandler.py

```diff
@@ -479,9 +498,134 @@ def checkD1File(self, submission_id=None):
 
         return JsonResponse.create(StatusCode.OK, response)
 
+    def loadGenerateRequest(self):
```

Comment:
Only thing there is that it looks a lot like our naming of interface functions to pull from DB.  I was hoping to make this clear that it's pulling from the http request, would something like getRequestParams make more sense, maybe getRequestParamsForGenerate since this only applies to the file generation routes?